hack/create-multikueue-kubeconfig.sh

```diff
@@ -126,15 +126,14 @@ CA_CERT=$(kubectl get -n ${NAMESPACE} secrets/${SA_SECRET_NAME} -o "jsonpath={.d
 # Extract cluster IP from the current context
 CURRENT_CONTEXT=$(kubectl config current-context)
 CURRENT_CLUSTER=$(kubectl config view -o jsonpath="{.contexts[?(@.name == \"${CURRENT_CONTEXT}\"})].context.cluster}")
-CURRENT_CLUSTER_ADDR=$(kubectl config view -o jsonpath="{.clusters[?(@.name == \"${CURRENT_CLUSTER}\"})].cluster.server}")
 
 echo "Writing kubeconfig in ${KUBECONFIG_OUT}"
 cat > ${KUBECONFIG_OUT} <<EOF
 apiVersion: v1
 clusters:
 - cluster:
     certificate-authority-data: ${CA_CERT}
-    server: ${CURRENT_CLUSTER_ADDR}
+    server: https://${CURRENT_CLUSTER/#kind-}-control-plane:6443
```

Comment:
This looks suspiciously hardcoded.
Are you sure this would work outside of mac os?

Reply:
It usually works , but for some reason after the worker reconnects and manager->worker connection works, the host -> worker client has a strange behavior. (getting a cluster queue takes abnormally long ).